doc/airline.txt

```diff
@@ -854,6 +854,18 @@ lsp <https://github.com/prabirshrestha/vim-lsp>
 * lsp close_lnum_symbol >
   let airline#extensions#lsp#close_lnum_symbol = ')'
 
+-------------------------------------                    *airline-nvimlsp*
```

Comment:
Alphabetical order, please.

(In alphabetical order, it should be below neomake.)